tests/manage/z_cluster/test_rook_operator_restart_during_mon_failover.py

```diff
@@ -77,7 +77,7 @@ def test_rook_operator_restart_during_mon_failover(self, node_drain_teardown):
         sample = TimeoutSampler(
             timeout=timeout, sleep=10, func=self.check_mon_pods_eq_3
```

Comment:
Looks like you missed to update this  line with the new func name, it still has the old function name.
Please change it to check_number_of_mon_pods